src/js/views/Search.js

```diff
@@ -12,16 +11,13 @@ import * as uiActions from '../services/ui/actions';
 import * as mopidyActions from '../services/mopidy/actions';
 import * as spotifyActions from '../services/spotify/actions';
 import { titleCase } from '../util/helpers';
+import { withRouter } from '../util';
 import { i18n } from '../locale';
 
 class Search extends React.Component {

```

Comment:
This whole component needs a good bit of spring cleaning. Overhauling the UX and rewriting the logic from scratch is high on my agenda.